source/js/components/join/join.jsx

```diff
@@ -262,23 +266,61 @@ export default class JoinUs extends React.Component {
    * Render the CTA heading.
    */
   renderSnippetHeading() {
-    return (
-      <React.Fragment>
-        <h5 className="h5-heading">
-          {!this.state.apiSuccess ? `${this.props.ctaHeader}` : `Thanks!`}
-        </h5>
-        {!this.state.apiSuccess ? (
+    let header = (
+      <Localized id="cta-header">{`Protect the internet as a global public resource`}</Localized>
+    );
+    let description = (
+      <Localized id="cta-description" strong={<strong></strong>}>
+        <p>{`Join our <strong>Mozilla News</strong> email list to take action and stay updated!`}</p>
+      </Localized>
+    );
+    let doubleOptIn = (
+      <div>
+        <Localized id="cta-double-opt-in" strong={<strong></strong>}>
+          <p>
+            {`If you haven’t previously confirmed a subscription to a Mozilla-related newsletter you may have to do so. <strong>Please check your inbox or your spam filter for an email from us.</strong>`}
+          </p>
+        </Localized>
+        <Localized
+          id="cta-manage-subscriptions"
+          elems={{
+            preferencesLink: (
+              <a
+                href="https://www.mozilla.org/newsletter/recovery/"
+                target="_blank"
+              ></a>
+            )
+          }}
+        >
```

Comment:
@stasm It turns out eslint is not a big fan of the new format, do you think there’s a workaround for that?
https://travis-ci.org/github/mozilla/foundation.mozilla.org/jobs/677437756#L295

Reply:
I don't think that's related to the `@fluent/react` 0.12 update. The lint complains about the `<a>` element being empty; it's empty because it expects content to be inserted from the translation. This behavior hasn't changed in 0.12. You can put some fallback content inside the `<a>`.